tests/specialops/tusewithoutexperimental.nim

```diff
@@ -0,0 +1,10 @@
+{.push experimental: "callOperator".}
+
+template `()`(a, b: int): int = a + b
+let x = 1
+let y = 2
+doAssert x.y == 3
+
+{.pop.}
+
+doAssert not compiles(x.y)
```

Comment:
```suggestion
doAssert not compiles(x.y)

import musewithoutexperimental # checks that import doesn't enable it
doAssert not compiles(x.y)
```

and add a file `tests/specialops/musewithoutexperimental.nim` containing `{.experimental: "callOperator".}`